### PR TITLE
:8ball: update workflow (thanks claude)

### PR DESCRIPTION
**Pull request by khrnchn**

### CODE REVIEW

File: .github/workflows/add_qrcode.yml [hunk 1]
```diff
@@ -79,8 +84,17 @@ jobs:
 
       - name: Create Pull Request
         run: |
-          gh pr create -B main -H ${{ steps.create_branch.outputs.branch }} --title ':memo: add new institution from #${{ github.event.issue.number }}' --body "closes \#${{ github.event.issue.number }}
-
-          ${{ github.event.issue.body }}"
+          pr_body="## new institution addition
+          
+          - institution Name: ${{ steps.extract.outputs.nameOfMasjid }}
```
Comment (khrnchn) [inline .github/workflows/add_qrcode.yml:89]:
```suggestion
          - institution name: ${{ steps.extract.outputs.nameOfMasjid }}
```

**Comment by khrnchn** [comment]
ok tak @khursani8

**Comment by khursani8** [comment]
Aku takut, old issue org submit hilang

**Comment by khrnchn** [comment]
> Aku takut, old issue org submit hilang, dia hanya guna from latest in case concurrent

gg aku salah merge pr, ingatkan ni pr add masjid hahaha

**Comment by khursani8** [comment]
Takpe bole revert commit tu

**Comment by khursani8** [comment]
As for conflict tu, bole fikir better structure utk append, ada contoh conflict tak?

**Comment by khrnchn** [comment]
1. create issue a
2. create issue b before merging pr a
3. merge pr a
4. pr b conflict, dia jadi masjid b and masjid a same id, and the order is masjid b dulu baru masjid a

so cara aku resolve aku letak balik masjid a ke atas, then masjid b as a new entry

**Comment by khursani8** [comment]
Ok so issue dia sbb id laa kan?

Kalau generate id based on issue number + 1000 ok? Ke mesti ikut susunan

**Comment by khrnchn** [comment]
tak sure lak kacau code ke tak, rasa macam tak. tapi conflict tu mula mula jelah masa buat issue banyak banyak haha, lepastu dah merge pr baru create new issue

**Comment by khursani8** [comment]
Kalau ko sorg bole control flow, nti ramai contribute takut conflict plak 🤣

**Comment by khursani8** [comment]
Dan aku igt nk tukar commiter tu instead of dari actions bot, nk tukar jd org yg submit

At least nama dia ada dkt contributer

**Comment by khursani8** [comment]
@khrnchn image

**Comment by khrnchn** [comment]
alright thanks sani satgi aku update

**Pull request merged**